substrate/client/network/sync/src/strategy.rs

```diff
@@ -176,6 +176,34 @@ pub enum SyncingAction<B: BlockT> {
 	Finished,
 }
 
+// Note: Ideally we can deduce this information with #[derive(derive_more::Debug)].
+// However, we'd need a bump to the latest version 2 of the crate.
```

Comment:
Why not do the bump?

Reply:
Will come with a follow-up, IIRC they have changed some feature-flags and now we have to explicitly select them :D